### PR TITLE
[doc] gtfs2ntfs: precision on line comments

### PR DESCRIPTION
**Pull request by papailio**
cf. ND-1075

### CODE REVIEW

File: documentation/gtfs_to_ntfs_specs.md [hunk 1]
```diff
@@ -188,7 +188,7 @@ _Warning :_ If the GTFS route has no trips, the Navitia Route should NOT be crea
...
 | routes.txt   | line_id        | Required   |            |                 | corresponding `line.id` (see Line construction above)                                                                                                       |
 | routes.txt   | destination_id | Optional   |            |                 | This field contains a stop_area.id of the most frequent destination of the contained trips (ie. the parent_station of the most frequent last stop of trips) |
-| comments.txt | comment_value  | Optional   | routes.txt | route_desc      | See (3) for additional properties                                                                                                                           |
+| comments.txt | comment_value  | Optional   | routes.txt | route_desc      | The comment is generated only when the parameter `read-as-line` deactivated. See (3) for additional properties                                                                                                                           |
```
Comment (woshilapin) [inline documentation/gtfs_to_ntfs_specs.md:191]:
```suggestion
| comments.txt | comment_value  | Optional   | routes.txt | route_desc      | The comment is generated only when the parameter `read-as-line` is deactivated. See (3) for additional properties                                                                                                                           |
```

Comment (pbougue) [inline documentation/gtfs_to_ntfs_specs.md:191]:
```suggestion
| comments.txt | comment_value  | Optional   | routes.txt | route_desc      | The comment is generated only when the parameter `read-as-line` is deactivated. See (3) for additional properties                                                                                                                           |
```

File: documentation/gtfs_to_ntfs_specs.md [hunk 2]
```diff
@@ -219,9 +219,15 @@ specified, the conversion should stop immediately with an error.
...
 | lines.txt | line_sort_order    | Optional   | routes.txt | route_sort_order |                                                                                                                                                                                                                                                                                  |
 | lines.txt | commercial_mode_id | Required   | routes.txt | route_type       | See "Mapping of route_type with modes" chapter (1).                                                                                                                                                                                                                              |
+| comments.txt | comment_value | Optional | routes.txt | route_desc | The comment is generated only when the parameter `read-as-line` activated. See (2) for additional properties. |
```
Comment (pbougue) [inline documentation/gtfs_to_ntfs_specs.md:222]:
```suggestion
| comments.txt | comment_value | Optional | routes.txt | route_desc | The comment is generated only when the parameter `read-as-line` is activated. See (2) for additional properties. |
```

Comment (woshilapin) [inline documentation/gtfs_to_ntfs_specs.md:222]:
```suggestion
| comments.txt | comment_value | Optional | routes.txt | route_desc | The comment is generated only when the parameter `read-as-line` is activated. See (2) for additional properties. |
```

**Review by pbougue — APPROVED** [review]
minor comments only

**Review by woshilapin — CHANGES_REQUESTED** [review]

**Review by woshilapin — APPROVED** [review]

**Comment by woshilapin** [comment]
Good for me but the commit has been integrated and will be merged through #741. I'm closing the current PR.

**Pull request closed without merging**